webapp-django/crashstats/authentication/management/commands/makesuperuser.py

```diff
@@ -10,7 +10,8 @@ def get_input(text):
 
 class Command(BaseCommand):
 
-    help = 'emailaddress [, otheremailaddress, ...]'
+    def add_arguments(self, parser):
+        parser.add_argument('emailaddress', nargs='+', type=str)
 
     def handle(self, *args, **options):
         if not args:
```

Comment:
I think this check is wrong. If you're using the `add_arguments` thing, then it uses argparse, so then the value is in `options['emailaddress']`.

This is the equivalent command in Fjord:

https://github.com/mozilla/fjord/blob/master/fjord/base/management/commands/ihavepower.py